connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -2065,6 +2065,43 @@ class PlanGenerationTestSuite
       fn.lit(Array(new CalendarInterval(2, 20, 100L), new CalendarInterval(2, 21, 200L))))
   }
 
+  test("function typedLit") {
+    simple.select(
+      fn.typedLit(fn.col("id")),
+      fn.typedLit('id),
+      fn.typedLit(1),
```

Comment:
If I remember correctly, `typedLit` in sql module support `Option` value , like `fn.typedLit(Some(1))`?  Is `typedLit` in connect client will also support it?

Reply:
Thank you for the reminder.